### ISSUE TITLE
Error compiling, missing dependencies - esx-base

### ISSUE THREAD

**Issue by s3luX** [VFrontDe/ESXi-Customizer-PS#41]
Hi there,

First off, I want to give you proprs for an awesome script that has helped us setting up home labs for years! 👍 

I reached a point where I need to reinstall my ESXi, and I wanted to make use of your script again.
I`m running Win10 with the PowerCLI version 5.5 and your latest script (v2.8.1) and when I try and run the script it throws me the following error:

```
PowerCLI C:\ESXI> **.\ESXi-Customizer-PS.ps1 -v67 -vft -load net-e1000e,net55-r8168,net55-r8169,net-r8101,net-r8139too,vmware-esx-dvfilter-maclearn,sata-xahci -pkgDir C:\ESXI\pkg -outDir C:\ESXI\ISO -log C:\ESXI\ISO\ESXI-custom.log -nsc**

This is ESXi-Customizer-PS Version 2.8.1 (visit https://ESXi-Customizer-PS.v-front.de for more information!)
(Call with -help for instructions)

Logging to C:\ESXI\ISO\ESXI-custom.log ...

Running with PowerShell version 5.1 and VMware vSphere PowerCLI 5.5 Release 1 build 1295336

Connecting the VMware ESXi Software depot ...Add-EsxSoftwaredepot : Could not download from depot at https://hostupdate.vmware.com/software/VUM/PRODUCTION/main/vmw-depot-index.xml, skipping (('https://hostupdate.vmware.com/software/VUM/PRODUCTION/main/vmw-depot-index.xml', '', '[Errno 4] IOError: <urlopen
error [Errno 1] _ssl.c:480: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>'))
At C:\ESXI\ESXi-Customizer-PS.ps1:250 char:22
+     if ($basedepot = Add-EsxSoftwaredepot $vmwdepotURL) {
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ReadError: (System.String[]:String[]) [Add-EsxSoftwareDepot], Exception
    + FullyQualifiedErrorId : ConnectDepotError,VMware.ImageBuilder.Commands.AddDepot


FATAL ERROR: Cannot add VMware ESXi Online depot. Please check your Internet connectivity and/or proxy settings!
```

I do not have any proxy in between the station and and vmware, so its not an issue of proxy. I have a strong feeling its something from my side, on the client config, but I cannot figure out what. I googled, and some posts say its due the openssl being out of date.

I know that SSL3 is not secure, and it is disabled by default on my PC, but I tried enabling it as well, but it didnt help, I tried running teh script via powershell as well, no luck there neither.

So I wanted to check if anyone else has encountered such an issue or maybe know the answer to it?

Thanks a lot in advance!

**Comment by s3luX** [comment]
Since I tried using an old PowerCLI (had it offline for years) and didnt download the latest one, the root cause was the outdated PowerCLI.

So for anyone having the same issues:
Use the latest PowerCLI with the latest SXi-Customizer-PS script and it will work.

**Comment by virtualex-itv** [comment]
most of the vibs you are attempting to load into the image are not compatible with vSphere 7.0 as they are linux-based and support for those types of drivers was removed in 7.0.  Please be sure to consult the vFront Online Depot and double check support for the drivers you are attempting to add.

I know that @VFrontDe has not updated the drivers for quite some time and none of the ones currently available are supported/compatible with ESXi 7.0.x.

**Comment by s3luX** [comment]
@virtualex-itv  Thanks for the clarification! Indeed, later on I tried with older versions of vibs and it worked. closing the case
